datascience-notebook/Dockerfile

```diff
@@ -4,23 +4,27 @@ LABEL maintainer=analytics-platform-tech@digital.justice.gov.uk
 
 USER root
 
-ENV PATH=$PATH:$HOME/.local/bin
-ENV NB_UID=1001
-ENV CHOWN_HOME=no
+ENV PATH=$PATH:$HOME/.local/bin \
+    NB_UID=1001 \
+    CHOWN_HOME=no
 
-COPY ./files/apt_packages /tmp/
```

Comment:
Remember to delete /files/apt_packages from the git index

Reply:
Already have thanks :)

Reply:
```
L0191:~/ap/analytics-platform-jupyter-notebook (testing/add-inspec)$ ls datascience-notebook/files/apt_packages
datascience-notebook/files/apt_packages
```
😕